index.js

```diff
@@ -430,6 +431,9 @@ module.exports = {
   get processColor(): processColor {
     return require('./Libraries/StyleSheet/processColor');
   },
+  get PlatformColor(): PlatformColor {
```

Comment:
Do we need to export the color functions for ios and android as well?

Reply:
yaass